### PR TITLE
Clean up SNB queries

### PR DESCRIPTION
**Pull request by aelred**

### CODE REVIEW

**Comment by aelred** [comment]
!rtg - redis didn't die properly

File: Jenkinsfile [hunk 1]
```diff
@@ -95,20 +95,21 @@ def buildGrakn() {
 }
 
 //Only run validation master/stable
-if (env.BRANCH_NAME in ['master', 'stable']) {
+// TODO: don't merge this
```
Comment (fppt) [inline Jenkinsfile]: I am once again conflicted

File: Jenkinsfile [hunk 2]
```diff
@@ -95,20 +95,21 @@ def buildGrakn() {
...
 
         slackGithub "Build started"
 
-        timeout(60) {
-            stage('Run the benchmarks') {
-                sh "mvn clean test --batch-mode -P janus -Dtest=*Benchmark -DfailIfNoTests=false -Dmaven.repo.local=${workspace}/maven -Dcheckstyle.skip=true -Dfindbugs.skip=true -Dpmd.skip=true"
-                archiveArtifacts artifacts: 'grakn-test/test-integration/benchmarks/*.json'
-            }
-        }
+//         timeout(60) {
```
Comment (fppt) [inline Jenkinsfile]: Ooooo commented code with no context

File: grakn-test/test-snb/src/main/java/ai/grakn/GraknQueryHandlers.java [hunk 3]
```diff
@@ -42,13 +39,47 @@
...
 
 import java.time.LocalDateTime;
-import java.time.ZoneOffset;
 import java.util.ArrayList;
 import java.util.Comparator;
 import java.util.List;
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
+import static ai.grakn.SNB.$content;
```
Comment (fppt) [inline grakn-test/test-snb/src/main/java/ai/grakn/GraknQueryHandlers.java]: :(

File: Jenkinsfile (hunk 2 above)
Comment (aelred) [reply Jenkinsfile]: Oh, this is just so I can see it pass on Jenkins :P
It will go before merge.

**Comment by aelred** [comment]
@fppt I removed the dumb thing 👍

**Pull request merged**